src/py/pyodide/_package_loader.py

```diff
@@ -22,8 +23,14 @@
 from ._core import IN_BROWSER, JsProxy, to_js
 
 SITE_PACKAGES = Path(getsitepackages()[0])
-STD_LIB = Path(sysconfig.get_path("stdlib"))
-TARGETS = {"site": SITE_PACKAGES, "lib": STD_LIB, "dynlib": Path("/usr/lib")}
+if sys.base_prefix != sys.prefix:  # Are we in a virtualenv?
```

Comment:
Maybe,
```suggestion
if sys.base_prefix != sys.prefix:  # We are in a virtualenv
```
because otherwise there is a doubt which branch corresponds to the virtualenv and it's not very obvious from the code.

Also here aren't the two branches inverted,  https://stackoverflow.com/a/1883251/1791279 ?

> The most reliable way to check for this is to check whether sys.prefix == sys.base_prefix. If they are equal, you are not in a virtual environment; if they are unequal, you are.

So they are unequal in the first branch, meaning we are in a venv, while we used this code before outside of it.

Reply:
Yes thanks. This is the reason for a bunch of the test failures.